WalletWasabi/WabiSabi/Backend/Rounds/Round.cs

```diff
@@ -13,6 +13,7 @@
 using WalletWasabi.WabiSabi.Crypto;
 using WalletWasabi.WabiSabi.Crypto.CredentialRequesting;
 using WalletWasabi.WabiSabi.Models;
+using MP = WalletWasabi.WabiSabi.Models.MultipartyTransaction;
```

Comment:
This is very unusual and should be used only when there are conflicts with types from other namespaces. Just use:

```suggestion
using WalletWasabi.WabiSabi.Models.MultipartyTransaction;
```

Reply:
ack, i had a conflict before but I think after enough renaming it went away but this stayed, I think it can be removed now

Reply:
although for the record, I find qualified imports more readable in general, especially when you don't have a prefix in the leaf-level type name. For example suppose we had 3 types of `Round` classes, I would rather see `Backend.Round`, `Client.Round`, `Model.Round` than just `Round` and infer which kind it is from context, which is still nicer than `BackendRound`, `ClientRound` and `ModelRound` or something like that.

Reply:
oh I remember what it was, there's barely a need to get hypothetical... `Parameters` vs. `RoundParameters`... it feels really wrong to name `MultipartyTransaction.Parameters` to `MultipartyTransaction.MultipartyTransactionParameters` and by the same logic arguably `RoundParameters` should be in a `Round` namespace